docs/src/managed/virtual-nodes-and-enhanced-clusters.md

```diff
@@ -0,0 +1,33 @@
+# OKE Enhanced Clusters and Virtual Nodes
+
+CAPOCI supports OKE [Enhanced Clusters][enhanced-cluster] and [Virtual Nodes][virtual-node-pool]. A cluster-template 
+`cluster-template-managed-virtual-node.yaml` with Enhanced Cluster and Virtual Node Pool has been released in 
+CAPOCI release artifacts which can be referred using  the flavor `managed-virtual-node` in `clusterctl generate` 
+command.
+
+> NOTE: Virtual Node Pool require Cluster API release 1.4.3 or later.
```

Comment:
Should probably call out CAPOCI version here as well

Reply:
yes, fixed